packages/terra-abstract-modal/src/_ModalContent.jsx

```diff
@@ -73,9 +78,20 @@ const defaultProps = {
   isScrollable: false,
   role: 'dialog',
   rootSelector: '#root',
+  trapFocus: false,
```

Comment:
That's a good point. I made this passive so consumers like NotificationDialog, which has its own [focus trap behavior](https://github.com/cerner/terra-framework/blob/17f77f704c1c644b99b3b251a862ea4497032c7a/packages/terra-notification-dialog/src/NotificationDialog.jsx#L170) would not be affected. This might require further investigation about if NotificationDialog's focus trap behavior meets the a11y standards. Also, if today's non-focus trapping behavior of AbstractModal is considered a bug, we probably wouldn't even want to allow for a setting of 'false' for the `trapFocus` prop, and thus there would be no need for a `trapFocus` prop at all, we'd just always trap focus.

Reply:
Yeah we can just remove the prop and always trap focus. Thanks Steven!